test/models/buildingModels/building_list_test.dart

```diff
@@ -18,11 +18,14 @@ void main() {
       BuildingList buildingList = new BuildingList();
 
       // Get number of lines from source
-      String linebreak = '\n';
-      int numLines = linebreak.allMatches(BuildingList.buildingInfo).length;
 
-      // Compare lengths
-      expect(buildingList.getListOfBuildings().length, numLines);
+      int loyolaBuidings = BuildingList.buildingInfo[0]['buildings'].length;
+      int sgwBuildings = BuildingList.buildingInfo[1]['buildings'].length;
+
+      int numOfBuildings = buildingList.getListOfBuildings().length;
+
+
+      expect(loyolaBuidings + sgwBuildings, numOfBuildings);
```

Comment:
this would be better if it was dynamic something along the lines of
```dart
int dynSum = 0;
for (int index = 0; BuildingList.buildingInfo.length; index++) {
    int length = BuildingList.buildingInfo[index]['buildings'].length;
    dynSum += length;
}

int numOfBuildings = buildingList.getListOfBuildings().length;
expect(dynSum, numOfBuildings);

Reply:
Oh yeah I missed that. I agree 👍